### PR TITLE
fix(console.log): add custom chrome formatters

### PR DESCRIPTION
**Pull request by smothers**
### Testing
- [ ] added relevant test coverage
- [x] no tests needed

### Docs
- [x] added relevant docs 
    - preview them at https://homebase.io/docs/homebase-react/{BRANCH_NAME}/overview
- [x] updated relevant sections in the README.md
- [ ] updated relevant docstrings in index.d.ts
- [ ] no docs needed

### Typescript
- [ ] added or edited relevant Typescript type declarations
- [x] no type declaration updates needed

## Merging
For maintainers.

To merge, select "Squash and Merge". Then:
  1. Make sure the top commit message follows [Angular Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines),
  2. Delete all other commit messages in the description, but keep any lines designating [co-authors](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors) so contributors will retain credit for their contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,30 @@ Check out the [Firebase example](https://homebaseio.github.io/homebase-react/#!/
 
 ##  Debugging tips
 
+### Custom chrome formatters
```
Comment (rubinovitz) [inline README.md:192]: @smothers this looks great. Can we add something like this to the documentation website as well?

File: src/dev/example/js/todo.jsx [hunk 2]
```diff
@@ -145,6 +145,7 @@ const TodoList = () => {
 // or sibling Todos don't trigger unnecessary re-renders.
 const Todo = React.memo(({ id }) => {
   const [todo] = useEntity(id)
+  console.log(todo)
```
Comment (rubinovitz) [inline src/dev/example/js/todo.jsx:148]: @smothers is leaving this `console.log` here on purpose?

**Review by rubinovitz — CHANGES_REQUESTED** [review]
Super exciting! Added a couple of comments.

**Comment by rubinovitz** [comment]
Also looks like a test is failing?

File: README.md (hunk 1 above)
Comment (smothers) [reply README.md]: just did. I thought it was pretty important so I stuck it on the front page (Overview)

File: src/dev/example/js/todo.jsx (hunk 2 above)
Comment (smothers) [reply src/dev/example/js/todo.jsx:148]: yes. I link here in the docs as an example

**Review by rubinovitz — APPROVED** [review]

**Pull request merged**